src/mds/Locker.cc

```diff
@@ -2991,6 +2992,10 @@ void Locker::_update_cap_fields(CInode *in, int dirty, MClientCaps *m, inode_t *
 	      << " for " << *in << dendl;
       pi->mtime = mtime;
     }
+    if (((dirty & CEPH_CAP_FILE_WR) && change_attr > pi->change_attr) ||
+	((dirty & CEPH_CAP_FILE_EXCL) && change_attr != pi->change_attr)) {
+      pi->change_attr = change_attr;
```

Comment:
I'm only thinking of doing an extra bump if multiple Ceph clients have indicated a change and bumped it.

I want to make sure the NFS client sees the right thing in the following scenario:

NFS Client -> Ganesha makes change resulting in change bumping from 4 to 5
NFS client does a GETATTR with change 5
On another Ceph client another change is made also makes a change, also bumping change from 4 to 5
Changes are consolidated on Ceph mds

Now if the mds finalizes everything with change 5, and the NFS client does a new GETATTR and sees change 5, it will assume there was no change, thus missing whatever changes occurred on the other Ceph client, whereas if the mds had bumped the change to 6, the NFS client would then know the file has changed beyond what it had already observed.